frontends/election-manager/src/components/election_manager_tally_report.tsx

```diff
@@ -0,0 +1,483 @@
+import React, { useContext, useEffect, useRef, useState, useMemo } from 'react';
```

Comment:
As the base for the new `ElectionManagerTallyReport` subcomponent, I copied the `TallyReportScreen` component.

So this commit is purely the result of:

```
cp frontends/election-manager/src/screens/tally_report_screen.tsx \
   frontends/election-manager/src/components/election_manager_tally_report.tsx
```

In the next commit, I actually whittle this file down to the `ElectionManagerTallyReport`